hack/allocate.sh

```diff
@@ -71,6 +71,8 @@ containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:50000"]
```

Comment:
@jrangelramos changes here might be of interest for you. We might get rid of `ttl.sh` in future.